.gitignore

```diff
@@ -1 +1,2 @@
-/venv
\ No newline at end of file
+/venv
+.idea
```

Comment:
you can find some default .gitignore file contents for most of languages, if not for all of them :)
the one for python: https://github.com/github/gitignore/blob/master/Python.gitignore